### PR TITLE
split pipeline into multible workflows

### PR DESCRIPTION
**Pull request by 6543**
…  to be able to be able to run on different platforms

### CODE REVIEW

**Comment by 6543** [comment]
well I would like to use matrix feature too ... but https://github.com/woodpecker-ci/woodpecker/issues/1870 right now is blocking me, to be usefull for lemmy ...

**Comment by 6543** [comment]
it fix  https://github.com/woodpecker-ci/woodpecker/issues/1677 ... kind a ;)

File: .woodpecker/test.yml [hunk 1]
```diff
@@ -1,19 +1,12 @@
-# TODO: The when: platform conditionals aren't working currently
-# See https://github.com/woodpecker-ci/woodpecker/issues/1677
+platform: linux/amd64
+
+when:
+  event: [pull_request, push]
```
Comment (6543) [inline .woodpecker/test.yml:4]: you need to decide if it should run if merged too ... else just remove push event ...

File: .woodpecker/publish.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+steps:
+  prepare_repo:
+    image: alpine:3
+    commands:
+      - apk add git
+        #- git fetch --tags
+      - git submodule init
+      - git submodule update
+
+  publish_release_docker_manifest:
```
Comment (6543) [inline .woodpecker/publish.yml:21]: manifest publish will need your eyes !!!
did not test it myselve jet!

**Comment by 6543** [comment]
PS: I think it's best if you would just fetch my commit and create your own pull ... where the main maintainers can work on ...

or ask and I'll add you to my fork so github do allow you to push to this pull branch :)

**Comment by dessalines** [comment]
Thx for this, just might be a few days till we can get to it.

**Comment by 6543** [comment]
No problem this is more a idea dump you can finalize ;)

**Comment by 6543** [comment]
Hope it helps !

**Comment by Nutomic** [comment]
I dont think we really need this as we dont use arm builders.

**Pull request closed without merging**